pkg/utils/common/controller.go

```diff
@@ -500,3 +499,16 @@ func (s *SuiteController) ApplicationGitopsRepoExists(devfileContent string) wai
 		return s.Github.CheckIfRepositoryExist(gitOpsRepoURL), nil
 	}
 }
+
+// CreateServiceAccount creates ServiceAccount by given name in a given namepsace
+func (s *SuiteController) CreateServiceAccount(name string, namespace string, secrets []corev1.ObjectReference) (*corev1.ServiceAccount, error) {
```

Comment:
you can do `s/name string, namespace string,/name, namespace string,/` if you choose